src/main/java/com/cinema_booking/booking/seat/dto/SeatMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.cinema_booking.booking.seat.dto;
+
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import com.cinema_booking.booking.room.dto.RoomMapper;
+import com.cinema_booking.booking.seat.model.Seat;
+
+@Mapper(uses = { RoomMapper.class }, componentModel = "spring")
```

Comment:
We can define mapstruct this way in maven, and then he will by default create spring beans. https://chrysanthium.com/spring-mapstruct-lombok

Reply:
hmm the solution from article work but i left 
<path>
  <groupId>org.projectlombok</groupId>
  <artifactId>lombok-mapstruct-binding</artifactId>
  <version>0.2.0</version>
</path>